### PR TITLE
added rainbird icon/logo

### PR DESCRIPTION
**Pull request by JonGilmore**
## Proposed change

added rainbird icons/logos

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/1075
- This PR fixes or closes issue: https://github.com/home-assistant/brands/issues/567

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/rainbird.markdown [hunk 1]
```diff
@@ -17,8 +16,10 @@ This `rainbird` integration allows interacting with [LNK WiFi](https://www.rainb
 
 There is currently support for the following device types within Home Assistant:
 
+- [Configuration](#configuration)
 - [Sensor](#sensor)
 - [Switch](#switch)
+- [Services](#services)
```
Comment (frenck) [inline source/_integrations/rainbird.markdown]: Configuration and services are not devices type. Please remove this unrelated change.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/rainbird.markdown (hunk 1 above)
Comment (JonGilmore) [reply source/_integrations/rainbird.markdown]: sorry about that, think my markdown linter snuck this in on me without me realizing it. I'll remove it.

**Review by frenck — APPROVED** [review]
Thanks! 👍
